aderyn_core/src/detect/mod.rs

```diff
@@ -14,4 +15,16 @@ macro_rules! capture {
     };
 }
 
+#[macro_export]
+macro_rules! capture_raw {
```

Comment:
Is there a way we can merge this logic into `capture` so we just have the one macro @TilakMaddy ?